task-management/src/main/webapp/vue-app/tasks-management/components/Project/ProjectAssigneeManager.vue

```diff
@@ -149,7 +149,7 @@ export default {
       }
     },
     reset() {
-      if (this.manager && !this.manager.length>0) { // In case of edit existing event
+      if (this.manager && this.manager.length>0) { // In case of edit existing event
```

Comment:
```suggestion
      if (this.manager && this.manager.length>0) {
```

Reply:
@ahamdi can you validate the PR if ok ?
Thanks